cmd/dumpling/main.go

```diff
@@ -59,6 +61,10 @@ func init() {
 
 	flag.StringVar(&outputDir, "output", defaultOutputDir, "Output directory")
 	flag.StringVar(&outputDir, "o", defaultOutputDir, "Output directory")
+
+	flag.StringVar(&consistency, "consistency", "none", "Consistency level during dumping: {none|flush|lock|snapshot}")
```

Comment:
The default should be "auto" which uses "snapshot" for TiDB and "flush" for MySQL.